backend/application/import_observations/parsers/cyclone_dx/parser.py

```diff
@@ -111,6 +112,11 @@ def _create_observations(  # pylint: disable=too-many-locals
                             data, components, component, metadata
                         )
 
+                        purl_type = None
```

Comment:
This section shall be placed in method `normalize_origin_component` of `application.core.services.observation.py`. Then it is executed always when there are changes in an observation, no matter where the change occurs.

Reply:
Done